### DISCUSSION TITLE
getting started with carla

### DISCUSSION THREAD

**Discussion by 2021Andrew** [carla-simulator/carla#7440] [General]
I am failing to install carla on my anaconda prompt, am using python version 3. 12.2

**Comment by Jinviz** [comment]
Do you use carla ver 0.9.15?

**Reply by 2021Andrew** [reply]
Am using carla  ver 0.0.9.9 i managed to find a python version for it but i cant manaully control it the vehicle cause the app is crashing maybe its because am using cpu, i need agent assisitances

**Comment by MattRoweEAIF** [comment]
Hi, at the moment, the CARLA API is only available for Python versions 3.8, 3.9 and 3.10. Please create a conda environment for one of these versions and try again, we aim to include more Python versions in the future.

**Reply by 2021Andrew** [reply]
i managed to find a version for the carla am using. but am using cpu and its crashing when i try to do the manuel control of the vehicle so as to teach it how to behaviour
